app/src/main/java/com/gathering/android/auth/signin/repo/ApiSignInRepository.kt

```diff
@@ -22,10 +24,17 @@ class ApiSignInRepository @Inject constructor(
                 response: Response<AuthorizedResponse>
             ) {
                 if (response.isSuccessful) {
-                    // TODO Save the JWT in shared pref to be used in future API calls
-                    onResponseReady(ResponseState.Success(response.body()))
-                } else {
-                    onResponseReady(ResponseState.SuccessWithError(response.body()))
+                    if(response.body()?.code == 200) {
```

Comment:
In the next version of the BE API we use only the http codes not custom codes in body.